routes/api/v1/foods.js

```diff
@@ -113,6 +114,22 @@ function validateRequest(req) {
   })
 }
 
+router.delete('/:id', async (req, res, next) => {
+  res.setHeader("Content-Type", "application/json");
+  try {
+    const food = await Food.findOne({where: {id: req.url.slice(1)}})
```

Comment:
Should be able to use 'req.params.id' instead of slicing the url.

Reply:
Good call. I went ahead and made the change.